code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -121,6 +121,11 @@
 
 	return ..()
 
+mob/living/simple_animal/examine(mob/user)
+	. = ..()
+	if(stat == DEAD)
+		. += "<span class='deadsay'>Upon closer examination, [p_they()] appear[p_s()] to be dead.</span>"
```

Comment:
I don't think `deadsay` is the right span for this, it's intended to be used for, well, deadsay. Unspanned is probably fine, or `notice` if you want it blue or `warning` if you want it red.

Reply:
@nemvar told me to use deadsay, make up your goddamn minds

Reply:
why in the world does human code use deadsay for this